tests/baselines/reference/emptyArrayBindingPatternParameter04.js

```diff
@@ -0,0 +1,11 @@
+//// [emptyArrayBindingPatternParameter04.ts]
+
+
+function f([] = [1,2,3,4]) {
```

Comment:
nothing wrong with the unused variable. we are guaranteed to have the variable name unique. i would rather have a single code path/pattern when possible.